pkg/appmgmt/appmgmt_recovery.go

```diff
@@ -28,18 +28,18 @@ import (
 	"github.com/apache/yunikorn-k8shim/pkg/appmgmt/general"
 	"github.com/apache/yunikorn-k8shim/pkg/appmgmt/interfaces"
 	"github.com/apache/yunikorn-k8shim/pkg/cache"
-	"github.com/apache/yunikorn-k8shim/pkg/common/utils"
 	"github.com/apache/yunikorn-k8shim/pkg/log"
 )
 
-func (svc *AppManagementService) WaitForRecovery(maxTimeout time.Duration) error {
+func (svc *AppManagementService) WaitForRecovery() error {
```

Comment:
Please document the expected behaviour for the function under normal and test case setup.
It looks like the abort could happen in production but the change explicitly says no timeout for app recovery

Reply:
Updated comments. The abort is only used by testing code.